webapp/components/Form.tsx

```diff
@@ -89,13 +91,15 @@ const Form = (): JSX.Element => {
       <Input
         name="Din kasserers e-post"
         value={mailto}
+        pattern="[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}"
         required
         updateForm={setMailto}
         helperText="Ofte 'gruppe-kasserer@ntnui.no'"
       />
       <Input
         name="Kontonummer"
         value={accountNumber}
+        pattern="^\d{4}\.\d{2}\.\d{5}$"
         required
         type="number"
```

Comment:
This wouldn't work as the type is set to `number` on the field, so we can't require periods 😅 

Also, we might have to reconsider this validation due to #27, as the account number needs to be Norwegian, **or** an international IBAN account number.

Reply:
Understand, that was just the format that is often send by norwegians and I, as european, find BBAN really confusing :D
If you want to add IBAN, that would make things more complicated, i would have to read up on the packet you linked in #27. 
So i will change this to just ^\d{11}$ for 11 digits and tackle IBAN later